scripts/webhook-patch-ca-bundle.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Original script found at: https://github.com/morvencao/kube-mutating-webhook-tutorial/blob/master/deployment/webhook-patch-ca-bundle.sh
+
+ROOT=$(cd $(dirname $0)/../../; pwd)
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export CA_BUNDLE=$(kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}' | base64 | tr -d '\n')
```

Comment:
minor comment: `base64 -w=0` disables line wrapping, so that you won't have to run `tr -d '\n'`

Reply:
Done